frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -266,6 +268,19 @@
             groupInner(filteredProposals),
         ),
     );
+
+    // if the messageIndex has changed but the chatId has not, scroll to the specified message
+    let previousChatId: ChatIdentifier | undefined = undefined;
+    $effect(() => {
+        if (
```

Comment:
So is Svelte clever enough to only trigger this when `pathState.messageIndex` or `app.selectedChatId` change?
As opposed to when `pathState` or `app` are updated?